stdlib/REPL/src/docview.jl

```diff
@@ -243,10 +243,12 @@ end
 
 function summarize(binding::Binding, sig)
     io = IOBuffer()
-    println(io, "No documentation found.\n")
     if defined(binding)
-        summarize(io, resolve(binding), binding)
+        binding_res = resolve(binding)
+        !isa(binding_res, Module) && println(io, "No documentation found.\n")
```

Comment:
Extra blank lines don't semm useful here
```suggestion
        !isa(binding_res, Module) && println(io, "No documentation found.")
```